includes/class-wc-stripe-webhook-handler.php

```diff
@@ -312,8 +312,13 @@ public function process_webhook_dispute( $notification ) {
 
 		$order->update_meta_data( '_stripe_status_before_hold', $order->get_status() );
 
-		/* translators: 1) The URL to the order. */
-		$message = sprintf( __( 'A dispute was created for this order. Response is needed. Please go to your <a href="%s" title="Stripe Dashboard" target="_blank">Stripe Dashboard</a> to review this dispute.', 'woocommerce-gateway-stripe' ), $this->get_transaction_url( $order ) );
+		$message = sprintf(
+		/* translators: 1) HTML anchor open tag 2) HTML anchor closing tag */
+			esc_html__( 'A dispute was created for this order. Response is needed. Please go to your %1$sStripe Dashboard%2$s to review this dispute.', 'woocommerce-gateway-stripe' ),
```

Comment:
This can be tested by creating an order, pay it with card, set it to processing again. And then use this postman endpoint to execute a webhook
[Stripe.postman_collection.json.zip](https://github.com/woocommerce/woocommerce-gateway-stripe/files/7629009/Stripe.postman_collection.json.zip)

Remember to change the charge to the one of the new order. You can find it in wp_post_meta with name _transaction_id. After that the string will show up as a note inside the order